code/game/jobs/job/assistant.dm

```diff
@@ -10,7 +10,7 @@
 	selection_color = "#dddddd"
 	access = list()			//See /datum/job/assistant/get_access()
 	minimal_access = list()	//See /datum/job/assistant/get_access()
-	alt_titles = list("Technical Assistant","Medical Intern","Research Assistant","Visitor")
+	alt_titles = list("Technical Assistant","Medical Intern","Research Assistant","Security Cadet", "Visitor")

```

Comment:
Not sure, it is as they are in the original SS13 code. Will test without them and report asap